### ISSUE TITLE
Landscape looks weird

### ISSUE THREAD

**Issue by birkir** [google-ar/three.ar.js#114]
Does three.ar.js only support portrait?

Landscape looks awfully bad for me

**Comment by jsantell** [comment]
no, all orientations are supported; do you have a screenshot of what you're seeing?
